vmm/src/config.rs

```diff
@@ -779,7 +779,8 @@ impl DiskConfig {
             .add("ops_refill_time")
             .add("id")
             .add("_disable_io_uring")
-            .add("pci_segment");
+            .add("pci_segment")
+            .add("serial");
```

Comment:
How does it appear to the guest? Is this name consistent with e.g lsblk?

Reply:
Unfortunately, the lsblk name is still `vda`, `vdb`, `vdc`, etc. That seems to be hardcoded by driver. It only changes affect any udev alias that includes the ID_SERIAL property.

So,
```
 ls -l /dev/disk/by-id/virtio-abcdefghijklmnopqrst
lrwxrwxrwx 1 root root 9 Sep  8 16:40 /dev/disk/by-id/virtio-abcdefghijklmnopqrst -> ../../vda
```

Reply:
It should show up under SERIAL column in lsblk.

```
$ lsblk -o NAME,SERIAL
 vda abcdefghijklmnopqrst
 ```

Reply:
This is what I was asking - how lsblk calls the field. I wanted us to match, so I think serial is a good choice.

Reply:
It also shows up in sysfs under serial.

```
cat /sys/block/vda/serial
abcdefghijklmnopqrst
```